src/main/resources/application-local.yml

```diff
@@ -31,7 +31,8 @@ case:
     case-service: http://localhost:8082
     workflow-service: http://localhost:8091
     document-service: http://localhost:8083
+    info-service: http://localhost:8085
     identity:
-      group: /TEST_TEAM_GROUP
-      team: 00000000-0000-0000-0000-000000000002
-      user: 00000000-0000-0000-0000-000000000003
+      group: /evPR7Fa9S063C6SLsHkNaw
+      team:
```

Comment:
As this service should work for either UKVI and migration, this should stay a sensible default and changed through usage. Alternatively create a local-ukvi and local-migration.

Reply:
AFAICT application-[profile].yml only matches a single profile so e.g. application-local-migration.yml wont work. Instead I added the ukvi/migration overrides using spring.config.activate.on-profile sections within application-local.yml. This setup should leave the config as it was before for the ukvi profile and override the queue and credentials for migration. It's working for me when testing.

Reply:
So we could also have a application-local, application-ukvi, and application-migration.  When specifying the profiles I believe it follows the order so we would specify `local ukvi` and it should read local and then override any values specified within ukvi. Think this is probably worth a go, if not, we'll park it for now.